lib/features/modeling/behavior/SubProcessStartEventBehavior.js

```diff
@@ -0,0 +1,80 @@
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import { is } from '../../../util/ModelUtil';
+import { isExpanded } from '../../../util/DiUtil.js';
+
+/**
+ * Add start event child by default when creating an expanded subprocess
+ * with create.start or replacing a task with an expanded subprocess.
+ */
+export default function SubProcessStartEventBehavior(eventBus, modeling) {
+  CommandInterceptor.call(this, eventBus);
+
+  eventBus.on('create.start', function(event) {
+    var shape = event.context.shape,
+        hints = event.context.hints;
+
+    if (
```

Comment:
Shouldn't this be something like hints.shouldAddStartEvent = is('subporcess') && isExpanded(shape)?

That piece of code will set shouldaddStartEvent for every task.

Reply:
Good point, that simplifies things.